src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -235,61 +261,78 @@ fields for the maps.
+| LINEARITY           | Tensor linearity (Westin 1997)                                     | DTI, DKI, WMTI (D, Dh, Dr)                                                                                                   |                                                                                                                        |
+| PLANARITY           | Tensor planarity (Westin 1997)                                     | DTI, DKI, WMTI (D, Dh, Dr)                                                                                                   |                                                                                                                        |
+| SPHERICITY          | Tensor sphericity (Westin 1997)                                    | DTI, DKI, WMTI (D, Dh, Dr)                                                                                                   |                                                                                                                        |
+| MK                  | Mean kurtosis                                                      | DKI, WMTI                                                                                                                    | Unitless                                                                                                               |
+| RK                  | Radial kurtosis                                                    | DKI, WMTI                                                                                                                    | Unitless                                                                                                               |
+| AK                  | Axial kurtosis                                                     | DKI, WMTI                                                                                                                    | Unitless                                                                                                               |
+| GFA                 | Generalized Fractional Anisotropy                                  | CSA, CSD, SHORE, MAPMRI, Forecast (or any model that can be represented as an ODF)                                           | Proportion \[0 - 1\]                                                                                                   |
+| INDEX               | Scalar maps from bedpostx                                          | FSUM (sum of partial volume fractions of the sticks)F_i (volume fraction of stick i)D (diffusivity)DSTD (std of diffusivity) | FSUM \[0 - 1\]F_i \[0 - 1\]D and DSTD microns^2/ms (for example, for free water in body temperature, this should be 3) |

```

Comment:
I see it was already like this before, but while we are, could you try to add some line breaks in the rendered version? Or, at least, `;` with spaces?

Reply:
This whole line is actually completely wrong: it's stating that "INDEX" is the model parameter being expressed, and then "`FSUM`" / "`F_i`" / "`D`" / "`DSTD`" are the models from which the "INDEX" parameter may be derived.

Reply:
@Lestropie thanks for pointing this out. Would you please clarify how you suggest changing the line?

Reply:
Resolved by 74ff6d0f; re-raise a new comment if you think the proposed solution is incomplete or not appropriate.